### DISCUSSION TITLE
KusionStack + KCL 社区中文周会 (2024.01.04)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#14] [General]
腾讯会议列表：778-2381-6338

**Comment by JameyFischer** [comment]
This weekly meeting was very practical and fostered a strong community atmosphere. The fact that KusionStack and KCL were discussed together is inherently valuable. Regular exchanges demonstrate an active and continuously growing community. This weekly meeting format is excellent. It doesn't need to be overly formal, but it effectively clarifies issues and shares experiences. It's beginner-friendly and beneficial for experienced members as well. Beyond the technical aspects, connecting people is crucial. The fact that everyone can sit in the same meeting, discuss practices, pitfalls encountered, and thought processes is invaluable. Players choose BitStarz for its smooth performance and wide range [bit-starz.net](https://bit-starz.net/) of entertaining games. I hope these weekly meetings continue and look forward to more people participating to strengthen the community.
